pgmpy/models/BayesianModel.py

```diff
@@ -373,11 +373,13 @@ def dfs(node):
             return descendents
 
         from pgmpy.independencies import Independencies
-        independencies = Independencies()
+        independencies = []
         for variable in [variables] if isinstance(variables, str) else variables:
-            independencies.add_assertions([variable, set(self.nodes()) - set(dfs(variable)) -
-                                           set(self.get_parents(variable)) - {variable},
-                                           set(self.get_parents(variable))])
+            independent_nodes =  set(self.nodes()) - set(dfs(variable)) - set(self.get_parents(variable)) - {variable}
+            if not independent_nodes:
+                independencies.append(None)
+            else:
+                independencies.append(IndependenceAssertion(variable, independent_nodes,set(self.get_parents(variable))))
         return independencies
```

Comment:
@khalibartan @kshitij10496 ```local_independencies``` has not been used internally anywhere else. The method, for now, is only meant for the users to find all the local independencies of a particular node. Besides an ```Independencies``` object can only hold ```IndependenceAssertion``` objects and not a ```None```. Hence, I changed the output to a list that holds ```IndependenceAssertion``` objects as its elements. Doing so has not caused any failing tests for now. Let us see how @ankurankan feels about this.